Docker Support

KayOhtie opened the pull request:
Adding a Dockerfile using the Alpine image. Currently this results in a 24.6 MB Docker image using Alpine and using native binaries when possible. Dockerfile finds latest binary build release from go-chromecast and downloads into image. Can change `GC_BUILD` build-arg at build time to build for a different platform such as a Raspberry Pi.

KayOhtie committed:
Cast groups can't youtube, also no - in unix var names, strip with sed

KayOhtie committed:
Dockerfile and example docker-compose.yaml

KayOhtie committed:
Dockerfile added, shrank using alpine inclusions

KayOhtie commented:
I'd accept #7 first before accepting this one, accidentally created my Docker branch with the other fix included.

nichobi commented:
Awesome, thank you so much for contributing! 
Would you mind perhaps adding it to the installation instructions in the README? Doesn't have to be much, just some basic instructions under a Docker subheading beneath the Arch one.

KayOhtie committed:
Added cleanup to slim image further

KayOhtie committed:
Added example environment variables for SBC

KayOhtie committed:
First draft of Docker installation instructions

KayOhtie committed:
Added environment variable details for Docker

KayOhtie committed:
Added details to require using 'host' network type for Docker

KayOhtie commented:
Updated with some information for installing and using Docker or Docker Compose, and added to the configuration section for adjusting environment variables for either.

nichobi committed:
Format Docker installation instructions

nichobi commented:
Perfect, merged. Again, thank you very much for your contributions!
